mappings/net/minecraft/util/ClickType.mapping

```diff
@@ -0,0 +1,3 @@
+CLASS net/minecraft/class_5536 net/minecraft/util/ClickType
+	FIELD field_27013 LEFT Lnet/minecraft/class_5536;
+	FIELD field_27014 RIGHT Lnet/minecraft/class_5536;
```

Comment:
What did you not use automapped names?

Reply:
primary and secondary seemed less meaningful than left and right when referring to a mouse